### PR TITLE
Docs- add hints for configuring behat locally

### PR DESCRIPTION
**Pull request by ptmkenny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This service can be used with any project type. The examples below are Drupal-sp
 ## Install/Update
 
 1. `ddev get drud/ddev-selenium-standalone-chrome`
-2. Optional. Update the provided .ddev/config.selenium-standalone-chrome.yaml as you see fit(and remove the #ddev-generated line). You can also just override lines in your .ddev/config.yaml
+2. Optional. Update the provided .ddev/config.selenium-standalone-chrome.yaml as you see fit (and remove the #ddev-generated line). You can also just override lines in your .ddev/config.yaml. Note that this installs the seleniarm image by default (for Apple Silicon machines), so if you are on x86 you probably want to use the [docker-selenium images](https://github.com/SeleniumHQ/docker-selenium) instead.
```
Comment (rfay) [inline README.md:10]: This is not correct. The image (https://hub.docker.com/r/seleniarm/standalone-chromium/tags?page=1&name=4.1.4-20220429) has both arm64 and amd64 versions, and the correct one is automatically pulled from hub.docker.com based on your architecture. The user does not have to be involved with thinking about architecture here at all.  This is true of all the images DDEV uses, they all have arm64 and amd64 pushes that make this possible.

```suggestion
2. Optional. Update the provided .ddev/config.selenium-standalone-chrome.yaml as you see fit (and remove the #ddev-generated line). You can also just override lines in your .ddev/config.yaml.
```

**Review by rfay — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ptmkenny) [reply README.md:10]: Wow, that's an awesome feature.  I mistakenly added this comment based on a reading of the seleniarm README without understanding how ddev works.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ This service can be used with any project type. The examples below are Drupal-sp
   - Drupal Test Traits
     - Ensure you have a working site that has the `weitzman/drupal-test-traits` Composer package.
     - `ddev exec -d /var/www/html/web "../vendor/bin/phpunit --bootstrap=../vendor/weitzman/drupal-test-traits/src/bootstrap-fast.php --printer '\Drupal\Tests\Listeners\HtmlOutputPrinter' ../vendor/weitzman/drupal-test-traits/tests/ExampleSelenium2DriverTest.php"`
+- Behat
+  - When setting up a local `behat.yml`, for the value of `wd_host`, use `http://selenium-chrome:4444/wd/hub`. For additional config values, see `config.selenium-standalone-chrome.yaml`.
+  - If your ddev site uses `https`, Chrome will complain about a self-signed certificate. This can be disabled by setting `Drupal\MinkExtension: sessions: javascript: selenium2: capabilities: chrome: switches: - '--ignore-certificate-errors'`.
```
[the remark this answers is not in the document]
Comment (ptmkenny) [reply README.md:29]: I think this is just how behat/mink behaves.  For example, this long-running feature request: https://github.com/Behat/MinkExtension/issues/47#issuecomment-192338092

The reason I added the note is that the name of the parameter to disable ssl varies by chrome version and there's no handy chart made by Google, so it's a bit of guesswork.  Having that noted in the README hopefully saves people the frustrating of searching and trying the wrong parameters and thinking the image is broken.

**Review by rfay — CHANGES_REQUESTED** [review]
Just a note, will leave the rest for the export @weitzman here.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (weitzman) [reply README.md:29]: Has something changed recently with Chrome? I have not needed to do any of this, and neither do our unit tests. By default ddev listens for http and https connections *inside the container*. Is there any chance you are running behat tests from outside the container?

[the remark this answers is not in the document]
Comment (rfay) [reply README.md:29]: @ptmkenny if this is an issue you can demonstrate in an example repo, maybe you could show how to create the problem step-by-step?  The intent of this add-on is that you shouldn't have to do any of these things.

[the remark this answers is not in the document]
Comment (ptmkenny) [reply README.md:29]: @weitzman @rfay Ok, I made a repo to demonstrate why this config is necessary: https://github.com/ptmkenny/d10-behat

This is the ddev d10 quickstart with behat and my minimal configuration/test added.

Steps to get running and see the issue:

```
ddev composer install
ddev drush site:install --account-name=admin --account-pass=admin -y
ddev composer behat
```

I'm not sure what causes me to have to use `--ignore-certificate-errors`, but if you comment out that line in `tests/behat/behat-ddev-selenium.yml` you will see that the test fails.

[the remark this answers is not in the document]
Comment (ptmkenny) [reply README.md:29]: @weitzman Thanks for sharing this config example.  I'll dig into it and see if I can find a better way for my own project.

**Comment by rfay** [comment]
@ptmkenny I hope you can check out https://github.com/drud/ddev-selenium-standalone-chrome/pull/17 and see if it works for you. Testing instructions provided there.

**Pull request closed without merging**